Cmake version 3.5.1does not work (git ff26b35 2018-11-06)

##  Behavior

cmake failed


## Steps to Reproduce the Problem

cmake -DCMAKE_BUILD_TYPE={Release,Debug}     -DHPX_WITH_MALLOC=system  ..

-- Performing Test HPX_WITH_RDTSCP - pre-set to TRUE
-- Boost version: 1.58.0
-- Found the following Boost libraries:
--   atomic
--   filesystem
--   program_options
--   regex
--   system
-- WARNING: HPX will perform poorly without tcmalloc or jemalloc. See docs for more info.
-- Using system allocator.
-- GIT commit is ff26b352be146122ae8d02feb72230b99c55f2f4.
CMake Error at examples/resource_partitioner/CMakeLists.txt:14 (if):
  if given arguments:
    "CMAKE_CXX_COMPILER_VERSION" "VERSION_GREATER_EQUAL" "5.0"

  Unknown arguments specified


-- Configuring incomplete, errors occurred!
See also "/home/aaa/hpx/hpx_git_x/bld/CMakeFiles/CMakeOutput.log".




## Specifications

... Please describe your environment

  - HPX Version:  git ff26b35 2018-11-06
  - Platform (compiler, OS):
gcc (Ubuntu 5.4.0-6ubuntu1~16.04.10) 5.4.0 20160609
Ubuntu 16.04 x86-64
cmake version 3.5.1

--- a/examples/resource_partitioner/CMakeLists.txt
+++ b/examples/resource_partitioner/CMakeLists.txt
@@ -11,7 +11,7 @@ set(example_programs
 )
 
 if(CMAKE_COMPILER_IS_GNUCXX AND HPX_WITH_GCC_VERSION_CHECK)
-  if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 5.0)
+  if(NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 5.0)
     if (HPX_WITH_CXX14_RETURN_TYPE_DEDUCTION)
       set(example_programs ${example_programs}
         async_customization

